pbnh/app/.ropeproject/objectdb

```diff
@@ -0,0 +1 @@
+�}q.
```

Comment:
Sorry about that, it's a refactor library cache that's automatically generated by vim. I removed it.